components/infobox/commons/infobox_league.lua

```diff
@@ -61,7 +61,7 @@ function League:createInfobox()
 				:cell('Server', args.server)
 				:fcell(Cell:new('Type')
 					:options({})
-					:content(args.type)
+					:content(args.type:sub(1,1):upper()..args.type:sub(2))
```

Comment:
Use `mw.language:ucfirst(s)` instead
https://www.mediawiki.org/wiki/Extension:Scribunto/Lua_reference_manual#mw.language:ucfirst

Reply:
Requires an init of Language right? Why do you prefer it?

Reply:
iirc mw.language:ucfirst(s) also works for `Ö, Ü, Ä`

Reply:
This is an emergency fix so I'm gonna merge this, and then will follow-up with a better solution soon